frontend/lib/resolution/can-pass.cpp

```diff
@@ -333,8 +333,14 @@ bool
 CanPassResult::canConvertCPtr(Context* context,
                               const Type* actualT,
                               const Type* formalT) {
-  if (actualT->isCPtrType()) {
+  if (auto actualPtr = actualT->toCPtrType()) {
     if (auto formalPtr = formalT->toCPtrType()) {
+      // Allow constness casts: an int* can be a const int*.
+      // In Chapel lingo, `c_ptr(int)` is passable to `c_ptrConst(int)`.
+      if (formalPtr->isConst() && !actualPtr->isConst() &&
+          formalPtr->eltType() == actualPtr->eltType())
+        return true;
```

Comment:
what if the eltTypes are not exactly the same, but can be passed? For example, I was expecting a `c_ptr(int(32))` actual to pass to a formal defined as `c_ptrConst(int(?w))`

Reply:
For instantiation, this is a good point -- I've ensured this works.

I don't think it should work for conversions (c_ptrConst(int(64)) <- c_ptr(int(32))), though.